docs/examples/notebooks/ShapeFactor.ipynb

```diff
@@ -157,14 +157,14 @@
      "output_type": "stream",
      "text": [
       "initialization parameters: [1.0, 1.0, 1.0]\n",
-      "parameters post unconstrained fit: [0.99981412 2.00002042 3.00006469]\n"
+      "parameters post unconstrained fit: [1.00004623 1.99998941 3.00000438]\n"
      ]
     },
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/jovyan/pyhf/src/pyhf/tensor/numpy_backend.py:173: RuntimeWarning: divide by zero encountered in log\n",
+      "/Users/kratsg/pyhf/src/pyhf/tensor/numpy_backend.py:334: RuntimeWarning: divide by zero encountered in log\n",
```

Comment:
As @kratsg and I talked about I'll strip the user file paths out, but this isn't a big dela.